test/integration/postOrder.spec.ts

```diff
@@ -49,7 +55,7 @@ suite("SDK: order posting", () => {
       accountAddress: walletAddress,
       amount: OFFER_AMOUNT ?? "0.004",
       quantity: 1,
-      paymentTokenAddress: WETH_ADDRESS,
+      paymentTokenAddress: WETH_ADDRESS_BY_NETWORK[Network.Main],
```

Comment:
Can be a separate PR but it might be nice to have Main be the default but let other networks run the test suite.